src/wagtail_personalisation/templates/modeladmin/wagtail_personalisation/segment/dashboard.html

```diff
@@ -29,6 +29,7 @@ <h2>{{ segment }}</h2>
                                             <ul class="inspect segment_stats">
                                                 <li class="stat_card">
                                                     {% trans "This segment has been visited" %}
+                                                    <!-- TODO: Change this icon -->
                                                     <span class="icon icon-fa-rocket">{{ segment.visit_count|localize }} {% trans "time" %}{{ segment.visit_count|pluralize }}</span>
```

Comment:
Hi @nickmoreton 
Any idea on how to use the icons? I already tried the following:
```suggestion
<!-- TODO: Change this icon -->
{% icon name="rocket" %}{{ segment.visit_count|localize }} {% trans "time" %}{{ segment.visit_count|pluralize }}
```

But it looks like:
![image](https://github.com/torchbox-forks/wagtail-personalisation/assets/26372762/b0db89f4-1e2a-40b5-9a06-51c482dca9e9)

I tried applying the class:
`{% icon name="rocket" classname="icon icon-rocket" %}`

But there are no styles being applied.

Reply:
I found this works:

```
<span>{% icon name="rocket" classname="w-w-4 w-h-4" %} {{ segment.visit_count|localize }} {% trans "time" %}{{ segment.visit_count|pluralize }}</span>
```

I found the classname options by inspecting the info button in the top bar of the HomePage listing and tried them.

```
<svg class="icon icon-info-circle w-w-4 w-h-4 group-hover:w-transform group-hover:w-scale-110" aria-hidden="true"><use href="#icon-info-circle"></use></svg>
```

Reply:
I also tried this for the calendar icon:

```
<span>{% icon name="calendar-alt" classname="w-w-4 w-h-4" %} {{ segment.enable_date|days_since:segment.disable_date }} {% trans "day" %}{{ segment.enable_date|days_since:segment.disable_date|pluralize }}</span>
```
But thats a built in wagtail icon so didn't need to add to the register_icons method. I wonder if there are enough suitable icons to be able to not even use wagtailfontawesomesvg